packages/components/src/box-control/vertical-horizontal-input-controls.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
```

Comment:
Not sure if using `lodash` in Gutenberg is a good practice — I know we're trying to reduce usage in other repos. (cc @tyxla)